regression-tests/mixed-bugfix-for-ufcs-non-local.cpp2

```diff
@@ -38,7 +38,7 @@ d: _ == t<o.f()>(); // Fails on Clang 12 (lambda in unevaluated context).
 
 u: @struct type = {
   b: bool == o.f();
-  c: bool == :(forward x: decltype(o.f())) = x;(true); // Fails on Clang 12 (lambda in unevaluated context).
```

Comment:
[Explanation] As mentioned in the PR description, the UFCS restored here in 299256076dfae69cf4badd6fa10150815e6a1715 makes all compilers report errors/crash on this tests.
For this reason I suggest to keep the regular call until the cause is identified and fixed.